<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>795</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SATKIT Annotator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="mplwindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplWindowVerticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="button_organiser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Go to Recording</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="goLineEdit">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="goButton">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="prevButton">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>Save Annotations...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="databaseView"/>
       </item>
       <item>
        <widget class="QWidget" name="ultrasoundFrame" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="positionRB">
         <property name="title">
          <string>Customised Metadata: TonguePosition</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="positionRB_1">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>High</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tonguePositionRBs</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="positionRB_2">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Low</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tonguePositionRBs</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="positionRB_3">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Other / Not visible</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tonguePositionRBs</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save_current_textgrid"/>
    <addaction name="action_save_all_textgrids"/>
    <addaction name="action_save_all"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_image">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionSave_plot_as"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menu_script">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Script</string>
    </property>
    <addaction name="actionShow_interpreter"/>
    <addaction name="actionRun_file"/>
   </widget>
   <widget class="QMenu" name="menu_navigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionNext_Frame"/>
    <addaction name="actionPrevious_Frame"/>
   </widget>
   <widget class="QMenu" name="menu_export">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="action_export_aggregate_images"/>
    <addaction name="action_export_annotations_and_metadata"/>
    <addaction name="action_export_distance_matrices"/>
    <addaction name="action_export_main_figure"/>
    <addaction name="action_export_ultrasound_frame"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_export"/>
   <addaction name="menu_image"/>
   <addaction name="menu_script"/>
   <addaction name="menu_navigation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_all">
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveToPickle">
   <property name="text">
    <string>Save to Pickle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save subplot as...</string>
   </property>
  </action>
  <action name="actionSave_plot_as">
   <property name="text">
    <string>Save plot as...</string>
   </property>
  </action>
  <action name="actionShow_interpreter">
   <property name="text">
    <string>Show interpreter</string>
   </property>
  </action>
  <action name="actionRun_file">
   <property name="text">
    <string>Run file...</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next Recording</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous Recording</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="action_export_analysis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export analysis...</string>
   </property>
  </action>
  <action name="actionNext_Frame">
   <property name="text">
    <string>Next Frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious_Frame">
   <property name="text">
    <string>Previous Frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_export_main_figure">
   <property name="text">
    <string>Export main figure...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_export_ultrasound_frame">
   <property name="text">
    <string>Export ultrasound figure...</string>
   </property>
  </action>
  <action name="action_export_annotations_and_metadata">
   <property name="text">
    <string>Export annotations and metadata...</string>
   </property>
  </action>
  <action name="actionfoobar">
   <property name="text">
    <string>foobar</string>
   </property>
  </action>
  <action name="action_export_aggregate_images">
   <property name="text">
    <string>Export aggregate images...</string>
   </property>
  </action>
  <action name="action_save_all_textgrids">
   <property name="text">
    <string>Save all TextGrids</string>
   </property>
  </action>
  <action name="action_save_current_textgrid">
   <property name="text">
    <string>Save current TextGrid</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_export_distance_matrices">
   <property name="text">
    <string>Export distance matrices...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tonguePositionRBs"/>
 </buttongroups>
</ui>
